src/pages/Aggregator/Aggregator.component.tsx

```diff
@@ -136,7 +142,6 @@ export const AggregatorComponent = ({
 
   useEffect(() => {
     resetField(AggregatorInputs.SendAmount);
-    resetField(AggregatorInputs.ReceiveAmount);
```

Comment:
Why did you remove it? Now you can end up in a state like this.
<img width="957" alt="image" src="https://user-images.githubusercontent.com/14878713/212039610-520f5a2b-8132-4288-8ce6-5d4ff3890edb.png">